### ISSUE TITLE
Fission Core without Helm - Missing Prometheus Endpoint in Deployment YAML

### ISSUE THREAD

**Issue by serain** [fission/fission#959]
If you install Fission without Helm, using the installation instructions:

```
$ kubectl apply -f https://github.com/fission/fission/releases/download/0.11.0/fission-core-0.11.0.yaml
```

The `controller` will loop crash. The logs are showing the `fission-bundle` "help" output, suggesting an invalid command:

```
 $ kubectl logs controller-7b79dbf768-mxv5n
Usage:
  fission-bundle --controllerPort=<port> --prometheusSvc=<url>
  fission-bundle --routerPort=<port> [--executorUrl=<url>]
  fission-bundle --executorPort=<port> [--namespace=<namespace>] [--fission-namespace=<namespace>]
  fission-bundle --kubewatcher [--routerUrl=<url>]
  fission-bundle --storageServicePort=<port> --filePath=<filePath>
  fission-bundle --builderMgr [--storageSvcUrl=<url>] [--envbuilder-namespace=<namespace>]
  fission-bundle --timer [--routerUrl=<url>]
  fission-bundle --mqt   [--routerUrl=<url>]
  fission-bundle --version
```

Digging through the `controller` deployment, the `controller` container is missing the Prometheus Service URL:

```
args: ["--controllerPort", "8888"]
```

Fixed by changing to:

```
args: ["--controllerPort", "8888", "--prometheusSvc", "http://fission-core-prometheus-server.default"]
```

NB: I also had to remove the version from all DNS/names due to #922/#939 - even though I'm not using Helm as the issue description suggests.

**Comment by soamvasani** [comment]
@smruthi2187 could you please take a look?

**Comment by smruthi2187** [comment]
Yes, it's a miss from my side. I've removed this parameter now. Fix  will be available in 0.12.0 release.

**Comment by soamvasani** [comment]
(This is fixed in #937 )
